tests/test_helpers.py

```diff
@@ -144,6 +144,22 @@
      "    aka:\n"
      "        -A. B. Friend\n        -AB Friend\n        -Tony Friend\n"
      ),
+    (["helper", "l_abstract", "run", "---author", "scopatz”,”--location”,”upton"],
```

Comment:
spaces after commas (see PEP8 for syntactic standards, but if you are using PyCharm you can have Pycharm modify the whole file automatically)

Reply:
yes just looked into some standards, for this, do you mean in particular to have a space after each comma as there is not one in the last two for location?

Reply:
yes.  If you are using PyCharm you can put the cursor right after that "bad" comma and you will see a yellow zigzag underscore.  A yellow light will pop up and you can click on it and it will say "reformat file" then you can do that.  It will make lots of changes in the file (formatting) so it is best to commit all your changes, then do this, then do another commit, or it becomes hard to find your actual edits alongside all the reformating edits